### DISCUSSION TITLE
Perfect!

### DISCUSSION THREAD

**Discussion by paulhypno** [valinet/ExplorerPatcher#558] [General]
Bravo! This fix is beyond perfect! I installed ExplorerPatcher because I mainly wanted to ungroup items on the Taskbar and get my right-click menu back. But I didn't want to have the old Windows 10 start menu (like what you get from StartAllBack I think). My start menu of choice is the excellent **Start MenuX**. 

Hey Presto! I have everything that I lost from upgrading to windows 11! No issues. I can now ungroup taskbar items, and everything else I lost from the 'upgrade', and there's a simple clear settings menu AND my **Start MenuX** with my start button of choice continues to work perfectly. 

A million thanks Valinet.

**Comment by valinet** [comment]
You're welcome, thanks for the kind words!
